transcom/mymove
# B 19895 tio queue payment requested int

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19895)

## Summary

This work is to show only moves with a status of Payment Requested in the TIO queue. Disabled the status dropdown as it is only one status now so no need to toggle/filter. Also moved a const from a `model_to_payload.go` file to `pkg/models/payment_request.go` to access that status in `queues.go`.

## To test
1. Log in as a TIO
2. See that you only see moves with a Payment requested status
3. Pick a move, remember it's locator. Take care of the pending payment request, which should change the status of the move
4. Go back to your TIO queue and verify that the move you just reviewed is no longer there.
5. Test that you can still search for a move of _any_ status as the TIO, as this work should not break that functionality.

## Screenshots
Before, showing moves with statuses other than Payment requested such as Deprecated, Error, 92 moves:
![image](https://github.com/transcom/mymove/assets/147537467/4b818285-6599-4ec9-93a3-467ad08a84d5)

After, only showing moves with Payment requested status, 77 moves:
![image](https://github.com/transcom/mymove/assets/147537467/0373f70b-266a-4a5f-93d5-4ab35adadadf)
